notifications_utils/polygons.py

```diff
@@ -168,7 +168,13 @@ def smooth(self):
         flattened = list(itertools.chain(*[
             flatten_polygons(polygon) for polygon in debuffered
         ]))
-        return Polygons(flattened)
+        return Polygons([
+            polygon for polygon in flattened if (
+                # The smoothing process creates some artifacts which can
+                # be removed by ignoring polygons less than 1m² in area
+                polygon.area > (1 / self.approx_square_metres_to_square_degree)
```

Comment:
should we have a test for this?

Reply:
Removing it will cause this bit in the admin app to raise an exception: https://github.com/alphagov/notifications-admin/blob/256c840b46d4095edbd0af105e5c1a340242575d/app/broadcast_areas/create-broadcast-areas-db.py#L140

It’s a bit hard to test those lines in isolation because they are part of a larger method. But I’ve added something which hopefully covers the expected behaviour in 36682a2